packages/bbui/src/Form/Core/Dropzone.svelte

```diff
@@ -94,6 +95,9 @@
       "change",
       value.filter((x, idx) => idx !== selectedImageIdx)
     )
+    await deleteAttachments(
```

Comment:
We need to wrap this in a `if (deleteAttachments) {...` as otherwise this will crash when the prop doesn't exist - such as anywhere within the builder.

Reply:
If statement added!